pkg/apis/agent/v1/klusterletaddonconfig_types.go

```diff
@@ -64,31 +67,47 @@ type KlusterletAddonConfigApplicationManagerSpec struct {
 
 	// +optional
 	ArgoCDCluster bool `json:"argocdCluster"`
+
```

Comment:
This technically should be removed in 2.4 @xiangjingli

Reply:
Yes, we no longer use the argocdCluster setting in klusterletAddonConfig for enabling/disabling importing ACM managed cluster secrets to argocd server since 2.3. It has been replaced by the new gitopsCluster CRD and controller.

@zhiweiyin318  Could you help remove the `ArgoCDCluster` field from the `klusterletAddonConfig` structure?

Reply:
will there be backward compatible issue here?

Reply:
It should be fine. See the argocdCluster setting is lost after upgrade to 2.4. the old argocdCluster controller watching the argocdCluster setting is also gone. So the existing ACM managed cluster secrets imported to the argocd server won't be deleted. If user want to import new ACM managed clusters, they will need to create the new gitopsCluster CRD.